src/pages/components/PageDetailsPage/PageDetailsPage.tsx

```diff
@@ -39,6 +42,8 @@ export interface PageDetailsPageProps {
   referenceProducts?: SearchProducts_search_edges_node[];
   allowEmptySlug?: boolean;
   saveButtonBarState: ConfirmButtonTransitionState;
+  selectedPageType?: PageType_pageType;
```

Comment:
How about

```suggestion
  focusedPageType?: PageType_pageType;
```

Reply:
I would stay with selected as it is indeed selected, not only focused.